llvm/include/llvm/ProfileData/Coverage/CoverageMapping.h

```diff
@@ -298,7 +298,7 @@ struct CounterMappingRegion {
                        unsigned ExpandedFileID, unsigned LineStart,
                        unsigned ColumnStart, unsigned LineEnd,
                        unsigned ColumnEnd, RegionKind Kind)
-      : MCDCParams(MCDCParams), ExpandedFileID(ExpandedFileID),
+      : MCDCParams(MCDCParams), FileID(FileID), ExpandedFileID(ExpandedFileID),
```

Comment:
You're correct.  I was thinking it might be useful to support ExpandedFileIDs in the future, but if that's the case, we can add it.  At the moment, I don't see when it would be useful anyway.